torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -306,3 +309,82 @@ def _create_onnx_friendly_decomposition_table() -> (
 DEFAULT_ONNX_EXPORTER_DECOMPOSITION_TABLE: Dict[
     torch._ops.OpOverload, Callable
 ] = _create_onnx_friendly_decomposition_table()
+
+
+def get_symbolic_function(
+    diagnostic_context: diagnostics.DiagnosticContext,
+    node: torch.fx.Node,
+) -> Union[onnxscript.OnnxFunction, Callable[..., Any]]:
+    if node.target == operator.getitem:
+        # __getitem__ on Tensor or Sequence of tensors. Not tuple.
+        exporter_key = "getitem"
+    elif (
+        isinstance(node.target, types.BuiltinFunctionType)
+        and node.target in _SYMINT_SYMFLOAT_BUILTIN_TO_EXPORTER_KEY_TABLE
+    ):
+        for node_arg in node.args:
+            if (not isinstance(node_arg, (torch.fx.Node, int, float))) or (
+                isinstance(node_arg, torch.fx.Node)
+                and not isinstance(node_arg.meta["val"], (torch.SymInt, torch.SymFloat))
+            ):
+                # TODO: reduce number of explicit initializations.
+                # TODO: Log location, stack.
+                diagnostic = diagnostics.UnsupportedFxNodeDiagnostic(
+                    diagnostics.rules.no_symbolic_function_for_call_function,
```

Comment:
Not sure if I understand this correctly. This PR registers two rules, but only `no_symbolic_function_for_call_function` is actually guarding errors. It seems the other one is prepared for some kind of full analysis, and it's not done yet? Which means it's not an actual rule to apply on converter code. It's rather a "wrapper rule" on the rules?

Reply:
Why do we need another layer to reveal unsupported node?